sql-plugin/src/main/scala/com/nvidia/spark/rapids/Plugin.scala

```diff
@@ -467,6 +467,9 @@ class RapidsDriverPlugin extends DriverPlugin with Logging {
             conf.shuffleTransportEarlyStartHeartbeatInterval,
             conf.shuffleTransportEarlyStartHeartbeatTimeout)
       }
+    } else {
+      logWarning("Rapids shuffle is not available, configured shuffle manager is " +
```

Comment:
I'm not sure this fixes the reported issue, because the issue was about the "Cannot initialize" message that's thrown in initShuffleManager.  This code is called when that's not, so the original issue will persist.  Also, do we want to log a warning every time the RAPIDS shuffle manager is not configured?  I see pros and cons, curious what others think.

Reply:
We were also discussing adding a check if the configured rapids manager does not match the spark (shim) version and warning (or should it be fatal?) on that.

Reply:
Yes, I thought that the original issue might have been fixed already. I suggested to double-check it and look at it with fresh eyes what we can improve in terms of actionable error message for common issues with this config.